packages/markdown/src/components/VoidElements/VoidElementsWarning.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { useVoidElements } from './useVoidElements';
+import { Note, Paragraph, TextLink } from '@contentful/forma-36-react-components';
+import tokens from '@contentful/forma-36-tokens';
+
+import { css } from 'emotion';
+
+const styles = {
+  note: css({
+    margin: `${tokens.spacingM} 0`
+  }),
+  paragraph: css({
+    '&:not(:last-child)': {
+      paddingBottom: tokens.spacingS
+    }
+  })
+};
+
+export function VoidElementsWarning() {
+  const { isEmpty } = useVoidElements();
+
+  if (isEmpty) {
+    return null;
+  }
+
+  return (
+    <Note noteType="warning" hasCloseButton={false} className={styles.note} testId="markdown-void-elements-warning">
+      <Paragraph className={styles.paragraph}>
+        Your markdown contains <TextLink linkType="primary" icon="ExternalLink" iconPosition="right">void
```

Comment:
Suggestion: Start the message with a TL;DR of the most important information: e.g. `Partial preview only`.
Then add some more details what causes this behaviour and how to fix it. @aromanko What do you think?

Where is the external link pointing to? I don't see a URL in the code

Reply:
@artibella I think "Your markdown contains void elements that causes preview to skip those parts" is pretty close to what you are suggesting. I'd like to get this merged and deployed soon and will come back to the copy if Anna wants a change of copy.

Reply:
About the copy: Ok for me to do it as you suggested

About the link URL: I am still confused where this external link is pointing to as I can not see it from the code. But maybe I am just missing something.

Reply:
@artibella it's pointing to https://www.w3.org/TR/2011/WD-html-markup-20110113/syntax.html#void-element

Reply:
Thanks @anho

Reply:
> Suggestion: Start the message with a TL;DR of the most important information: e.g. `Partial preview only`.
> Then add some more details what causes this behaviour and how to fix it. @aromanko What do you think?
> 
> Where is the external link pointing to? I don't see a URL in the code

"Partial preview only" sounds good to me, it can be formatted as heading and will provide the reader with some extra context.